CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Features ✨ and improvements 🏁
+* Update mapbox styles to latest versions. ([1837](https://github.com/mapbox/mapbox-maps-android/pull/1837))
```

Comment:
afair we were planning to release that in v10.10? then it should move to rc section I assume